Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
```

Comment:
Did we decide that we wanted to pull from the base Ubuntu image here rather than a pre-existing Keen image?